include/envoy/local_info/local_info.h

```diff
@@ -37,7 +37,7 @@ class LocalInfo {
   virtual const std::string& nodeName() const PURE;
 
   /**
-   * v2 API Node protobuf. This is the full node identity presented to management servers.
+   * This is the full node identity presented to management servers.
```

Comment:
`@return the full node identity presented to management servers.`

Reply:
Thanks for this. Updated.